Deploy-CAM.ps1

```diff
@@ -2101,6 +2220,7 @@ if ($ResourceGroupName) {
         New-AzureRmResourceGroup -Name $ResourceGroupName -Location $location
     } 
     $rgMatch = Get-AzureRmResourceGroup -Name $ResourceGroupName
+    $rgName = $rgMatch.ResourceGroupName
```

Comment:
Do we still need it? I thought I nuked the requirement for rgName (probably confusion because of base branch change.)

Reply:
ill double check, i added it back just to be safe

Reply:
looks like we can nuke it easy, will do